src/tree/transformer.ts

```diff
@@ -210,3 +212,11 @@ const getDependencies = (instance): Array<Dependency> => {
     decorators: parameterDecorators[i] ? parameterDecorators[i].map(d => d.toString()) : [],
   }));
 };
+
+const getQuickViewAttribute = (element): Array<Property> => {
+  if (!element.isComponent) {
+    return quickViewAttribute(element);
+  } else {
```

Comment:
More of a style, but you don't need and else clause. If you remove it, it is cleaner to read, just simply return an empty array.